### PR TITLE
Additional detail about Centralized Mail transport

### PR DESCRIPTION
**Pull request by JuanGasca**
The article says "you can route all mail from mailboxes in Exchange online" but the emails coming form internet will be routed to on-premises as well, if centralized mail flow is enabled. I've added the additional information there. 

Technical reference: 
https://docs.microsoft.com/en-us/exchange/transport-routing#inbound-messages-from-the-internet

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/transport-options.md [hunk 1]
```diff
@@ -37,10 +37,11 @@ You'll need to choose how to route inbound and outbound mail when you plan and c
 
 - Do you want to route outbound mail to external recipients from your Exchange Online organization through your on-premises organization (centralized mail transport), or do you want to route it directly to the Internet?
 
-    With centralized mail transport, you can route all mail from mailboxes in the Exchange Online organization through the on-premises organization before they're delivered to the Internet. This approach is helpful in compliance scenarios where all mail to and from the Internet must be processed by on-premises servers. Alternately, you can configure Exchange Online to deliver messages for external recipients directly to the Internet.
+    With centralized mail transport, you can route all mail from mailboxes in the Exchange Online organization through the on-premises organization before they're delivered to the Internet. The same way, incoming Internet messages will be routed to On-premises organization before being delivered to any Exchange On line recipient. Possible scenarios are detailed in Transport Routing in Exchange Hybrid deployments.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/transport-options.md:40]:
```suggestion
    With centralized mail transport, you can route all mail from mailboxes in the Exchange Online organization through the on-premises organization before they're delivered to the Internet. In the same way, incoming Internet messages will be routed to an on-premises organization before being delivered to any Exchange Online recipient. Possible scenarios are detailed in [Transport routing in Exchange hybrid deployments](https://docs.microsoft.com/microsoft-365/compliance/sensitive-information-type-entity-definitions?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**